src/api/index.ts

```diff
@@ -0,0 +1,41 @@
+import axios from "axios";
+import urljoin from "url-join";
+import { Column } from "../components/Board";
+
+const baseUrl = "http://localhost:3000";
+
+export default async function getData(): Promise<Column> {
+  const { data } = await axios.get(urljoin(baseUrl, "columns?_embed=tasks"));
+
+  return data;
```

Comment:
Wrong type. Should be `Promise<Column[]>`
or `Promise<Array<Column>>`

Reply:
if I assign a type other than Column, I get an error :
'Argument of type 'Column[]' is not assignable to parameter of type 'Column'.
 Type 'Column[]' is missing the following properties from type 'Column': id, title, tasks  TS2345'